internal/view/app.go

```diff
@@ -691,6 +693,54 @@ func (a *App) helpCmd(evt *tcell.EventKey) *tcell.EventKey {
 	return nil
 }
 
+// previousView returns to the view prior to the current one in the history
+func (a *App) previousView(evt *tcell.EventKey) *tcell.EventKey {
+	if evt != nil && evt.Rune() == rune(ui.KeyB) && a.Prompt().InCmdMode() {
+		return evt
+	}
+	cmds := a.cmdHistory.List()
+	if !(len(cmds) > 1) {
```

Comment:
A dialog here feels a bit heavy, I think best to use a flash for this instance.
Also not keen on !(len(cmds) > 1.  This is len(cmds) <=1 return. 
But... as noted below better to defer these calls to cmdHistory.

Reply:
Additionally all these should be on cmdHistory apis. ie the logic for toggle last view and fwd/backwd should be methods on the history and not view logic imho i.ie toggleLastCmd, prevCmd, nextCmd, etc..

Reply:
Additionally if we surfaced a cmdHistory view all the necessary apis would be already there.